specs/lib/commentOnClosedIssueSpec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fsExtra = require('fs-extra');
+var Promise = require('bluebird');
+var requestPromise = require('request-promise');
+
+var getUniqueMatch = require('../../lib/getUniqueMatch');
+
+var commentOnClosedIssue = require('../../lib/commentOnClosedIssue');
+
+describe('commentOnClosedIssue', function() {
+    it('throws if `url` or `message` is undefined', function () {
+        expect(function() {
+            commentOnClosedIssue();
+        }).toThrowError();
```

Comment:
Normally we would use `toThrowDeveloperError` for this, but we'll have to wait until we add the Jasmine matchers stuff to be able to do it, I only mention it so you know about it.